### ISSUE TITLE
[suggestion] Config error checking if crashing.

### ISSUE THREAD

**Issue by Darkmega18** [Glitchfiend/ToughAsNails#550]
Something like a line indication of where it cracked the shits, or what illegal value you stuck in would be nice. rather than crashing outright and having to guess what could be wrong or right on a large pack that takes a few minutes to load each time. :V

**Comment by Forstride** [comment]
Not really sure if it's possible to pinpoint to the specific line in an error message for this, but JSON files need to be formatted properly to load at all.  You likely left out a comma, or included one where it shouldn't be, or didn't format your brackets properly.  If we didn't cause the game to crash, most people likely wouldn't even notice, and their mistakes would lead to them telling us that the config system doesn't work because the file couldn't load at all.

**Comment by Darkmega18** [comment]
nope. just nope. I throw my json into an online json editor and it works fine when I shrink out the spacing then re-expand it. it shoots an error if I miss a comma, stuff up a bracket or what. so now I'm painstakingly going through 1k lines of drink configs to check which quarter of the file I fucked something up in if it'll tell me.

I know i never screwed up in the juice drinks I did the other night cause it loaded prior to me doing the last of my drinks over the last hour and a half. but then I improved their values and the hydration and then kept going...

and i'm not saying don't crash. i'm saying crash and like fathertoast could for stuff like mob properties back in the day, have it tell me what line it's null pointing due to something it doesn't know about. :V 

it'd crash then the crashlog would have a line literally telling me the line and pos of what I did illegal/where it hit a brick wall or missed formatting so I know where to look, and it's usually bleedingly obvious for the most part once you're told... was a god send working on special armored/named mobs full of nested arrays for enchants etc in mob properties on my first modpack ages back. :V 

and oh hey, there it goes. so it's somewhere in the relatively small amount of teas and milkshakes. fun times. [_*grumbles something about meaning to be in bed by now*_]
